contents/founders/first-1000-users.md

```diff
@@ -0,0 +1,229 @@
+---
+date: 2024-06-19
+title: 'How we got our first 1,000 users'
+author:
+  - james-hawkins
+  - andy-vandervell
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/math_meme_8b0013533e.jpg
+featuredImageType: full
+tags:
+  - Growth
+crosspost:
+  - Blog
+  - Newsletter
+---
+
+> This post is the first in a series where I’ll break down how we reached important milestones at PostHog.
+> 
+> - **Part 1:** How we got our first 1,000 users **(You are here)**
+> - **Part 2:** How we grew from 1,000 to 10,000 users (coming soon)
+> - **Part 3:** How we scaled to 100,000 users (coming soon)
+
+One of my favorite weeks as CEO of PostHog was an [offsite in Aruba](https://posthog.com/blog/aruba-hackathon), but it wasn't for the reason you expect. 
+
+We chose Aruba because the team deserved to celebrate. We’d grown massively, so we increased the offsite budget and opted for a fancy hotel in the Caribbean as a reward for everyone’s hard work.
+
+But I felt a little guilty when we arrived. 
+
+The lobby had a boat in it. There was a private beach. You could drink infinite piña coladas. I knew we were spending $100k more than we needed to and, as a founder, a core part of my job is making sure we don't run out of money!
+
+As any founder should, though, I:
+
+1. Compartmentalized my feelings (they don't add to shareholder value)
+2. Kept refreshing our revenue dashboard (TikTok told me manifesting is real)
+
+![posthog aruba](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/aruba/aruba-hackathon-photos.jpeg)
+
+It worked. Within two days, our annual recurring revenue (ARR) had grown enough to pay for the entire trip. Suddenly, I didn't feel guilty at all.
+
+This clearly demonstrated to me the efficiency of our business model – a model that’s propelled us from 10 users to 100,000 in just over four years. Today, around 1,200 companies – ranging from hobbyists to Fortune 500s – sign up every week. 
+
+This series is about sharing how we reached this point. It’s also about personal reflection – writing is thinking, after all. I’ll share the lessons I learned, and things I wish we’d done differently. 
+
+If you have a question, please leave a comment – I’ll try and answer them all. [My DMs on X are open](https://x.com/james406), too.
+
+## 🎉 Reaching 100 users in four weeks 
+
+Getting our first 10 users was the hardest part.
+
+Tim, my co-founder, and I wound up [pivoting five times](https://posthog.com/founders/story-about-pivots) before we landed on PostHog. We found it very hard to get 10 companies to use our products, even when we thought we were solving a real problem!
+
+When we pivoted to PostHog, we created a one-month plan. 
+
+The goal? Find out if anyone cared about the problem (self-hosted product analytics) as quickly as possible.
```

Comment:
```suggestion
Getting our first 10 users was the hardest part.

Tim, my co-founder, and I wound up [pivoting five times](https://posthog.com/founders/story-about-pivots) before we landed on PostHog. We found it very hard to get 10 companies to use our products, even when we thought we were solving a real problem!

When we pivoted to PostHog, we created a one-month plan. 

The goal? Find out if people disliked the existing product analytics tools enough to try ours instead, and do so as quickly as possible.
```

@ivanagas thoughts on this instead? I feel like I was presuming too much knowledge, but I don't want to spend a bunch of time explaining the idea. I think this is enough context at this stage.

Reply:
yeah, I think that is good